src/main/java/org/prebid/server/privacy/gdpr/model/TcfContext.java

```diff
@@ -29,9 +29,10 @@
 
     String ipAddress;
 
-    List<String> warnings;
+    @Builder.Default
```

Comment:
Nice approach, but I would like not to have it, because it adds logic that is completely is not obvious outside this class